### ISSUE TITLE
Compatibility to Vite (Angular 17)

### ISSUE THREAD

**Issue by stephanrauh** [stephanrauh/ngx-extended-pdf-viewer#1998]
Version 19.0.0-alpha.0 doesn't compile with Angular 17 because Vite stumbles upon an `await import` statement. This statement isn't used in ngx-extended-pdf-viewer, but it's in the binaries. Let's remove it.

**Comment by Tweniee** [comment]
can you provide more info?

**Comment by stephanrauh** [comment]
I've run into this error message:
<img width="987" alt="image" src="https://github.com/stephanrauh/ngx-extended-pdf-viewer/assets/3045767/7bcf0412-b709-4ade-ae38-45be441cf5d6">

**Comment by stephanrauh** [comment]
OK. I've pushed the code and I've even published the broken version yesterday (version 19.0.0-alpha.0). Only after adding the code to a fresh project I noticed the problem.

The code is the main branch of these two projects:
https://github.com/stephanrauh/ngx-extended-pdf-viewer
https://github.com/stephanrauh/pdf.js

My current approach is to comment code like this:
<img width="603" alt="image" src="https://github.com/stephanrauh/ngx-extended-pdf-viewer/assets/3045767/cd4f54fe-d7c0-4fa8-a73e-6419a2296b05">
The code using `await import` is never used in an Angular application, but it seams Vite doesn't know this, so it starts complaining.
